pkg/apis/kubeflow/validation/validation.go

```diff
@@ -35,7 +35,8 @@ var (
 
 	validMPIImplementations = sets.NewString(
 		string(kubeflow.MPIImplementationOpenMPI),
-		string(kubeflow.MPIImplementationIntel))
+		string(kubeflow.MPIImplementationIntel),
+		string(kubeflow.MPIImplementationMPICH))
```

Comment:
Should we update https://github.com/kubeflow/mpi-operator/blob/fda0532ba1a25e48b68316759c2dec438cfc8d13/pkg/apis/kubeflow/validation/validation_test.go as well?

Reply:
I'm yet waiting for this.

Reply:
Should be in now. I tried to refactor into a common function, so both Intel and MPICH tests just call it. As the part of that refactoring "valid with worker" tests uses has different restart policy, then it had before. However, I'm fine with that as it's not what's being tested.

Reply:
Thanks. Let me check.